src/main/java/com/coveo/pushapiclient/StreamOperationType.java

```diff
@@ -0,0 +1,16 @@
+package com.coveo.pushapiclient;
+
+public enum StreamOperationType {
+  REBUILD("rebuild"),
+  INCREMENTAL("incremental");
```

Comment:
Could it be clearer if the names here aligned with [the docs](https://docs.coveo.com/en/l62e0540/coveo-for-commerce/how-to-update-your-catalog#leading-practices)?
`Rebuild` is used in brackets [here](https://docs.coveo.com/en/lb4a0344/coveo-for-commerce/stream-your-catalog-data-to-your-source#:~:text=during%20the%20stream-,(rebuild),-process%20if%20all), but is `incremental` referenced?

I see references "full item updates" and "partial item updates" so maybe it's more consistent to use such names?